unit-tests/py/rspy/acroname.py

```diff
@@ -203,18 +204,28 @@ def enable_ports( ports = None, disable_other_ports = False, sleep_on_change = 0
     return result
 
 
-def disable_ports( ports ):
+def disable_ports( ports = None, sleep_on_change = 0 ):
     """
-    :param ports: List of port numbers
+    :param ports: List of port numbers; if not provided, disable all ports
+    :param sleep_on_change: Number of seconds to sleep if any change is made
     :return: True if no errors found, False otherwise
     """
     global hub
     result = True
-    for port in ports:
-        #
-        action_result = hub.usb.setPortDisable( port )
-        if action_result != brainstem.result.Result.NO_ERROR:
-            result = False
+    changed = False
+    for port in all_ports():
+        if ports is None or port in ports:
+            if is_port_enabled( port ):
+                    #log.d("disabling port", port)
+                    action_result = hub.usb.setPortDisable( port )
+                    if action_result != brainstem.result.Result.NO_ERROR:
+                        result = False
+                        log.e("Failed in disabling port", port)
```

Comment:
Failed to disable
But OK

Reply:
Will rename and add to `enable_ports` too

Reply:
Done